lms/templates/ccx/schedule.underscore

```diff
@@ -13,7 +13,7 @@
       <% _.each(chapters, function(chapter) { %>
         <tr class="chapter collapsed" data-location="<%= chapter.location %>" data-depth="1">
           <td class="unit">
-            <a href="#"><i class="fa fa-caret-right toggle-collapse"></i></a>
+            <a href="#" class="toggle-collapse"><i class="fa fa-caret-right"></i></a>
```

Comment:
Needs rebase. Done this in #9114 
https://github.com/edx/edx-platform/pull/9114/files#diff-f15570680848e2d23dbfad465e79372dR26